README.md

```diff
@@ -9,6 +9,7 @@ Airq allows you to recieve information about the air quality near you simply by
 
 Airq aggregates sensor readings from nearby [Purpleair](https://docs.google.com/document/d/15ijz94dXJ-YAZLi9iZ_RaBwrZ4KtYeCy08goGBwnbCU/edit?usp=sharing) sensors. These sensors are updated every 10 minutes so the results should be fairly accurate, generally. We will soon have a publicly available number to which you can text your zipcode to recieve air quality metrics.
```

Comment:
@wdanfort can you remove the line here where it says "We will soon have a publicly available number to which you can text your zipcode to recieve air quality metrics."? 

You just need to make an update to your branch, run `git add -A`, and then `git commit -m "<you message>"`, and then `git push`.

Reply:
think it's updated using this methodology here?